src/ZendeskApi.Contracts/Models/Organization.cs

```diff
@@ -49,8 +49,8 @@ public class Organization : IZendeskEntity
         [IgnoreDataMember]
         public long group_id { get; set; }
 
-        [IgnoreDataMember]
-        public long external_id { get; set; }
+        [DataMember(Name = "external_id", EmitDefaultValue = false)]
```

Comment:
If you make the change in the same branch as the one you created this PR from (fix-external-id-type) and commit the change, then it should be automatically added here.